src/V3SchedAcyclic.cpp

```diff
@@ -99,13 +99,13 @@ class Graph final : public V3Graph {
         // TODO: 'typeName' is an internal thing. This should be more human readable.
         if (LogicVertex* const lvtxp = dynamic_cast<LogicVertex*>(vtxp)) {
             AstNode* const logicp = lvtxp->logicp();
-            std::cerr << logicp->fileline()->warnOther()
+            std::cerr << logicp->fileline()->warnOtherGuarded()
```

Comment:
Maybe function name is a little bit misleading here, both functions ``warnOther`` and ``warnOtherGuarded`` are guarded, but the difference is what function locks the V3Error mutex.

In ``warnOther`` variant, the mutex is locked by ``v3warn`` macro and in ``warnOtherGuarded`` variant, this function itself locks V3Error mutex. I've done this to allow usage of ``warnOther`` outside of ``v3warn`` macro (``warnOther`` function underneath uses ``V3Error::warnMore``: https://github.com/verilator/verilator/blob/master/src/V3FileLine.cpp#L412).

Flipping the name of function would result a lot of changes in the code, every place where we are using ``fileline()->warnOther()`` in the code inside ``v3warn`` macro would need to be adapted. On the other hand there are only 2 cases where it is used outside of ``v3warn`` macro.

Reply:
I've added ``VL_MT_UNSAFE`` annotation to ``warnOtherGuarded`` function, so when it will be used in ``MT_SAFE`` context, ``clang_check_annotations`` script will warn about this.

Reply:
Maybe call this warnOtherStandalone then?